tsconfig.json

```diff
@@ -28,7 +28,11 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "esnext"
+    "target": "esnext",
+    "types": [
+      "vite-svg-loader",
+      "vite/client"
```

Comment:
what did adding these fix?

Reply:
Just TS being big mad when working in the vite config and refusing to do any type validation and also allows us to use `import.meta.hot` without adding it to our own meta definition